docs/maps/connect-to-ems.asciidoc

```diff
@@ -3,17 +3,56 @@
 
 :ems-docker-repo: docker.elastic.co/elastic-maps-service/elastic-maps-server-ubi8
 :ems-docker-image: {ems-docker-repo}:{version}
+:ems-headers-url: https://deployment-host
 
 https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
 tile layers and vector shapes of administrative boundaries.
 If you are using Kibana's out-of-the-box settings, Maps is already configured to use EMS.
 
+[float]
+=== Domains
+
 EMS requests are made to the following domains:
 
 * tiles.maps.elastic.co
 * vector.maps.elastic.co
 
-Maps makes requests directly from the browser to EMS.
+
+[float]
+=== Headers
+
+Find below examples of the request and response headers from Kibana and a minimal `curl` request example showing the response headers sent by each service.
+
+WARNING: This headers may change without further notice at anytime and are shared for reference.
+
+[float]
+==== EMS Tile Service
```

Comment:
How about a quick description of the service. Maybe something like  "The tile service returns base map vector tiles."